Dockerfile-debian

```diff
@@ -0,0 +1,9 @@
+FROM ubuntu:20.04
```

Comment:
Should it be debian image instead of ubuntu?

Reply:
Ubuntu is based on Debian so no problem in leaving this to me. What do you think @manu0466?

Reply:
Yeah, Ubuntu is based on Debian, but I am not sure that all Ubuntu features are also available on Debian. If no, this file should be renamed into `Dockerfile-ubuntu`.

Reply:
Since we are using it just to run the Desmos binary I don't see any issue using Ubuntu instead of Debian.
Also the `ubuntu:20.04` image is smaller then `debian:bullseye-slim`, ~69Mb for Ubuntu vs ~76MB of Debian.